openeo_udp/worldcereal_crop_extent.json

```diff
@@ -1291,27 +1320,13 @@
       }
     },
     {
-      "name": "orbit_state",
-      "description": "The orbit state of the Sentinel-1 data",
+      "name": "projection",
+      "description": "The EPSG code of the output projection",
```

Comment:
best to specify that UTM is the default, users should know that they do not need to set this value.